experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1450,6 +1507,13 @@ def _validate_feature_configs(self, data):
                 reference_branch_errors.append({"value": schema_errors})
                 continue
 
+            if fml_errors := self._validate_schema_against_fml(
+                feature_value_data["feature_config"],
+                data.get("channel"),
+            ):
```

Comment:
You don't want to be interacting with the schema at all. The schema is not the FML, its a generated JSON schema.